src/Stratis.Bitcoin.LightD/FullNodeBuilderExtensions.cs

```diff
@@ -0,0 +1,59 @@
+using System.Collections.Generic;
+using Microsoft.Extensions.DependencyInjection;
+using NBitcoin;
+using NBitcoin.Rules;
+using Stratis.Bitcoin.Base;
+using Stratis.Bitcoin.Builder;
+using Stratis.Bitcoin.Configuration.Logging;
+using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.Consensus.Rules;
+using Stratis.Bitcoin.Features.Consensus;
+using Stratis.Bitcoin.Features.Consensus.CoinViews;
+using Stratis.Bitcoin.Features.Consensus.Rules;
+using Stratis.Bitcoin.Interfaces;
+
+namespace Stratis.Bitcoin.LightD
+{
+    public static class FullNodeBuilderExtensions
+    {
+        public static IFullNodeBuilder UseLightPowConsensus(this IFullNodeBuilder fullNodeBuilder)
+        {
+            LoggingConfiguration.RegisterFeatureNamespace<PowConsensusFeature>("lightpowconsensus");
+
+            fullNodeBuilder.ConfigureFeature(features =>
+            {
+                features
+                    .AddFeature<PowConsensusFeature>()
+                    .FeatureServices(services =>
+                    {
+                        services.AddSingleton<ConsensusOptions, ConsensusOptions>();
+                        services.AddSingleton<DBreezeCoinView>();
+                        services.AddSingleton<ICoinView, CachedCoinView>();
```

Comment:
I think the light node does no need some of these interfaces (like DBreezeCoinView)

Reply:
I thought that CachedCoinVIew had a dependency on DBreezeCoinView to flush

Reply:
Yes, but light node doe not have a UTXO db, so all the rules that build the UTXO set are not needed.

Reply:
So should I remove ICoinView registration completely

Reply:
yes, and remove the rules that use it.